packages/docs/pages/StatefulTable/StatefulTablePage.tsx

```diff
@@ -223,6 +223,30 @@ const StatefulTablePage = () => {
         />
         {/* jsx-to-string:end */}
       </CodePreview>
+
+      <H2>Usage with search</H2>
+
+      <CodePreview scope={{ items }}>
+        {/* jsx-to-string:start */}
+        {function Example() {
+          return (
+            <StatefulTable
+              itemName="Products"
+              columns={[
+                { header: 'Sku', hash: 'sku', render: ({ sku }) => sku },
+                { header: 'Name', hash: 'name', render: ({ name }) => name },
+                { header: 'Stock', hash: 'stock', render: ({ stock }) => stock, sortKey: 'stock' },
+              ]}
+              items={items}
+              pagination
+              selectable
```

Comment:
Should we include this prop for a search example?
```suggestion
```

Reply:
Not necessary. I can remove it